spec/checkout_sdk/issuing/testing_issuing_integration_spec.rb

```diff
@@ -7,21 +7,39 @@
     before(:all) do
       @cardholder = create_cardholder
       @card = create_card @cardholder.id, true
+      @transaction = simulate_transaction @card
     end
 
     describe '.simulate_authorization' do
       context 'when simulating authorization with valid card' do
         it { authorizes_transaction get_issuing_api.issuing.simulate_authorization get_authorization_request }
       end
     end
+
+    describe '.simulate_increment' do
+      context 'when simulating increment authorization with valid card' do
+        it { authorizes_transaction get_issuing_api.issuing.simulate_increment @transaction.id, { 'amount' => 300 } }
+      end
+    end
+
+    describe '.simulate_clearing' do
+      context 'when simulating a clearing of an existing transaction' do
+        it { authorizes_transaction get_issuing_api.issuing.simulate_clearing @transaction.id, { 'amount' => 100 } }
+      end
+    end
+
+    describe '.simulate_reversal' do
+      context 'when simulating a reversal of an existing transaction' do
+        it { authorizes_transaction get_issuing_api.issuing.simulate_reversal @transaction.id, { 'amount' => 100 } }
+      end
+    end
   end
 end
 
 private
 
 def authorizes_transaction(response)
```

Comment:
clearing response doesn't have status response and reversal doesn't have `Authorized` status only `Reversed` or `Declined` status, please review.